docs/integrations/symfony2.rst

```diff
@@ -37,28 +37,25 @@ Capturing context can be done via a monolog processor:
 
 .. sourcecode:: php
 
-    namespace Acme\Bundle\AcmeBundle\Monolog;
+    namespace AppBundle\Monolog;
 
-    use Symfony\Component\DependencyInjection\ContainerInterface;
-    use Acme\Bundle\AcmeBundle\Entity\User;
+    use AppBundlee\Entity\User;
+    use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
```

Comment:
1. `AppBundlee` :smile_cat: 
2. Better to use `Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface` surely? (Don't forget to update the type hint on line 49 too!)